fuel-p2p/src/behavior.rs

```diff
@@ -134,3 +168,20 @@ impl NetworkBehaviourEventProcess<PeerInfoEvent> for FuelBehaviour {
         }
     }
 }
+
+impl NetworkBehaviourEventProcess<GossipsubEvent> for FuelBehaviour {
```

Comment:
Can we use American English? Defer to a future PR to avoid polluting this one.

Reply:
nope since that's what we use from libp2p, not my choice 🇬🇧

Reply:
🤷

Reply:
I mean, we could name our own FuelBehavior, but it would be weird since it would still need to implement libp2p's Behaviour